tensorboard/components/tf_data_selector/tf-data-select-row.ts

```diff
@@ -151,6 +146,16 @@ Polymer({
 
     const url = tf_backend.getRouter().runsForExperiment(this.experiment.id);
     return requestManager.request(url).then(runs => {
+      this._defaultRunStorageValue =
```

Comment:
As discussed, probably better to keep the initializer up in attached() and then implement this default value down in _getRunsSelectionState().